birdwatcher/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM python:3.11-bullseye
+FROM python:3.11-slim-bullseye
```

Comment:
3.12 doesn't work here?

Reply:
No, the aiohttp library still doesn't work with Python 3.12. They're on it. Probably in some weeks/month, we can migrate to 3.12.